shaptools/hana.py

```diff
@@ -447,6 +447,197 @@ def get_sr_status(self):
         status["status"] = SrStatusReturnCode.get(result.returncode, SrStatusReturnCode.UNKNOWN)
         return status
 
+    def set_ini_parameter(
```

Comment:
@Simranpal The difference of this method and the unset_ini_parameter is basically the `UNSET` keyword. That said, I would create some internal method that handles `SET` and `UNSET` via some parameter and then have set_ini_parameter and unset_ini_parameter as the public methods (interfaces) that calls this internal method.

This avoids all the duplication we have between these two methods.

@arbulu89 what do you think?

Reply:
Hi @diegoakechi , yes I was initially doing one method, there is another difference, ie the hana ini parameters: set takes param+their values and unset only takes params.

We can add logic for that too, right now im taking dictionary and list correspondingly for ini parameters. So I added two methods for now.

Reply:
Ok, get it working first and then we can consider to refactor this after that.

Reply:
@Simranpal Definetely, I would like to see this functions in one. 
What we can do, as the request different entry parameters is let them as public, but have an extra private method doing the rest, basically, the code is exactly the same.

The difference would be the parameters list `str` and `set/unset` keyword. 

So we could have something like `def _manage_ini_file(self, ..., set_value=true)` where all the same code is placed